sdk/keyvault/azure-security-keyvault-keys/inc/azure/keyvault/keys/cryptography/encrypt_parameters.hpp

```diff
@@ -56,12 +72,6 @@ namespace Azure {
     {
     }
 
-    /**
-     * @brief Encrypt Parameters can't be default constructed.
-     *
-     */
-    EncryptParameters() = delete;
```

Comment:
I am sure this is fine, but help me understand why we are removing this?

Reply:
The whole point of using EncryptParameters and DecryptParameters as factories was to help guide customers down a path of success. Different algorithms require different inputs and forbid use of others, either by convention or mandate, like no "iv" (nonce in that case) for AES-GCM by the Microsoft crypto board. Hence, EncryptParameters and DecryptParameters should not be default-creatable.

Reply:
> Hence, EncryptParameters and DecryptParameters should not be default-creatable.

Makes sense. So, @vhvb1989 keep the explicitly deleted default ctor then (if it meets the 0-3-5 rule from the guidelines)?